### ISSUE TITLE
Phone auth check if user exist

### ISSUE THREAD

**Issue by aideslucas** [RaphaelJenni/FirebaseUI-Angular#62]
Hi, I know the default behavior of firebase is that if the user doesn’t exists it creates a new one,
Is there a way I can get the phone number provided in the FirebaseUI-angular before it requests the code to send it to a function that checks if the user exists?

**Comment by RaphaelJenni** [comment]
At the moment this isn't possible, as all the data is inside the widget and not accessible from outside.
But what exactly is the use case?
If you just want to check if the user is a new one or not to have a custom app behavior, you can use the `additonalUserInfo` with the `isNewUser` flag. 
This is provided by firebaseui threw the authResult:
- https://github.com/firebase/firebaseui-web#available-callbacks
- https://firebase.google.com/docs/reference/js/firebase.auth#.UserCredential
- https://firebase.google.com/docs/reference/js/firebase.auth.html#.AdditionalUserInfo

**Comment by aideslucas** [comment]
My use case is that we don’t want to allow new users to be created from the login
So I would like to check if the user exists before firebase sends the verification code

**Comment by RaphaelJenni** [comment]
Found this on the firebaseui repo: https://github.com/firebase/firebaseui-web/issues/99

TL;DR: It is currently not supported, but the ticket is still open.

**Comment by RaphaelJenni** [comment]
I will support it as soon as firebase(ui) supports it.
Close the issue for now.
